ruff_lsp/server.py

```diff
@@ -455,11 +726,11 @@ class LegacyFix(TypedDict):
 )
 async def code_action(params: CodeActionParams) -> list[CodeAction] | None:
     """LSP handler for textDocument/codeAction request."""
-    document = LSP_SERVER.workspace.get_text_document(params.text_document.uri)
+    document = Document.from_uri(params.text_document.uri)
 
     settings = _get_settings_by_document(document.path)
 
-    if utils.is_stdlib_file(document.path):
+    if document.is_stdlib_file():
```

Comment:
You can also provide notebook specific code-actions. These are prefixed by namespace `notebook.` in code action kind.